src/utils/config.js

```diff
@@ -0,0 +1,5 @@
+let KEY = {
+  CAPTCHA_KEY: '6Lfl0G0UAAAAAOvzQPfpTKQFguY14Cn7w6fNvRx1',
```

Comment:
@fragm3 are you using the same captcha key used on chat.susi or is it different from it. If it's the same key then fetch it from the server as done on chat.susi and let me know if its different so that I can add it on the server.

Reply:
@PrP-11 should we use same KEY or different for both?